app/controllers/dashboard/trainer_controller.rb

```diff
@@ -45,4 +34,14 @@ def review_card
   def trainer_params
     params.permit(:user_translation)
   end
+
+  def next_card
+    if current_user.current_block
```

Comment:
So, you have some duplication here. 
How you can improve it without adding new methods?